OptionSelector/CQCategory.h

```diff
@@ -0,0 +1,17 @@
+//
+//  CQCategory.h
+//  Glycemic Index of Common Foods OptionSelector App
+//
+//  Created by Ayuna Vogel on 8/10/15.
+//  Copyright (c) 2015 Ayuna Vogel. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface CQCategory : NSObject
```

Comment:
(By the way, this comment syntax, with the two stars at the front, is called [Doxygen](https://en.wikipedia.org/wiki/Doxygen) and is really rich and has many many features, many of which I don't know about, but if you want to dig into it there's a lot there.)